src/platform-includes/sourcemaps/upload/esbuild/javascript.mdx

```diff
@@ -0,0 +1,58 @@
+Sentry uses [releases](/product/releases/) to match the correct source maps to your events. You can use the Sentry esbuild plugin to automatically create releases and upload source maps to Sentry when bundling your app.
+
+## Install
+
+```npm
+npm install @sentry/esbuild-plugin --save-dev
+```
+
+```yarn
+yarn add @sentry/esbuild-plugin --dev
+```
+
+## Configure
+
+Learn more about the configuration of the plugin using our [Sentry esbuild Plugin documentation](https://github.com/getsentry/sentry-javascript-bundler-plugins/tree/main/packages/esbuild-plugin).
+
+Example:
+
+```javascript {filename:esbuild.config.js}
+const sentryEsbuildPlugin = require("@sentry/esbuild-plugin");
+
+require("esbuild").build({
+  entryPoints: ["./src/index.js"],
+  outdir: "./dist",
+  bundle: true,
+  sourcemap: true, // Source map generation must be turned on
+  plugins: [
+    // Put the Sentry esbuild plugin after any other plugins
+    sentryEsbuildPlugin({
+      org: "___ORG_SLUG___",
+      project: "___PROJECT_SLUG___",
+
+      // Specify the directory containing build artifacts
+      include: "./dist",
+
+      // Auth tokens can be obtained from https://sentry.io/settings/account/api/auth-tokens/
+      // and needs the `project:releases` and `org:read` scopes
+      authToken: process.env.SENTRY_AUTH_TOKEN,
+
+      // Optionally uncomment the line below to override automatic release name detection
+      // release: process.env.RELEASE,
+    }),
+  ],
+});
+```
+
+The Sentry esbuild plugin will automatically inject a release value into the SDK so you must either omit the `release` option from `Sentry.init` or make sure `Sentry.init`'s `release` option matches the plugin's `release` option exactly:
```

Comment:
This is a good note + snippet, which IMHO should be on the other bundlers' pages that don't have it, too.